portfolio/src/main/webapp/script.js

```diff
@@ -61,26 +66,27 @@ function addRandomFact() {
 function showBlogPost(postNum) {
   // Find the correct triangle to expand and switch its image.
   // Also change the onclick function for this button to hide the post.
-  const buttonId = "expandTriangle" + postNum;
-  let button = document.getElementById(buttonId);
+  const buttonId = 'expandTriangle' + postNum;
+  const button = document.getElementById(buttonId);
   console.log(button);
   if (button !== null) {
-    button.src = "/images/downtriangle.jpeg";
+    button.src = '/images/downtriangle.jpeg';
     button.onclick = function() {
       hideBlogPost(postNum);
     };
   }
 
   // Add blog post content to the correct area.
-  const blogPostId = "blogPostArea" + postNum;
-  let postArea = document.getElementById(blogPostId);
+  const blogPostId = 'blogPostArea' + postNum;
+  const postArea = document.getElementById(blogPostId);
   if (postArea !== null) {
     if (postNum > 0 && postNum <= posts.length) {
-      const elements = posts[postNum-1]
-      for (i = 0; i < elements.length; i++) {
-        const element = elements[i]
+      const elements = posts[postNum - 1];
+      let i = 0;
```

Comment:
nit: it's generally better to declare these within the scope of the for loop, for clarity:

for (let i = 0; i < ........)